mesh-federation/templates/istio.yaml

```
@@ -21,6 +21,8 @@ spec:
       multiCluster:
         clusterName: {{ $.Values.localCluster }}
       network: {{ $.Values.localCluster }}-network
+      logging:
+        level: "all:debug"
```

Debug logs are noisy, so I would prefer to make it optional:
```suggestion
      {{- if $.Values.debug }}
      logging:
        level: "all:debug"
      {{- end }}
```